textalyze.js

```diff
@@ -14,10 +6,27 @@
 function itemCounts(array) {
   let counts = new Map();
 
-  // Your code here.
-  // Run 'npm test' to see what tests need to pass.
+  array.forEach(element => {
+    
+    if (!counts.has(element))
+      counts.set(element, 1);
+    else 
+      counts.set(element, counts.get(element) + 1);
+  });
 
   return counts;
 }
 
 module.exports = { itemCounts };
+
+//
+// Running...
+//
+
+var array = ['a', 'a', 'a', 'b', 'b', 'c'];
+
+console.log("The counts for ", array, " are...");
+
+itemCounts(array).forEach(function(value, key) {
+  console.log(key, '   ', value);
+});
```

Comment:
Make sure you have newlines at the end of your file.  `git` will complain if you don't and there are good reasons to do it (I can say more if you want).  Configure your editor so that it adds newlines to the end of files automatically.